crates/swift-bridge-ir/src/parse/parse_struct.rs

```diff
@@ -302,13 +320,33 @@ mod tests {
         assert_eq!(ty.swift_name.as_ref().unwrap().value(), "FfiFoo");
     }
 
+    /// Verify that we parse the derive(...)
+    #[test]
+    fn parse_derive_attribute() {
+        let tokens = quote! {
+            #[swift_bridge::bridge]
+            mod ffi {
+                #[derive(Copy, Clone)]
+                struct Foo;
+            }
+        };
+
+        let module = parse_ok(tokens);
+
+        let ty = module.types.types()[0].unwrap_shared_struct();
+        
+        let derives: Vec<String> = ty.derives.clone().unwrap().iter().map(|derive| derive.to_string()).collect();
+        assert_eq!(derives, vec!["Copy", "Clone"]);
+    }
+
     /// Verify that we properly parse multiple comma separated struct attributes.
     #[test]
     fn parses_multiple_struct_attributes() {
         let tokens = quote! {
             #[swift_bridge::bridge]
             mod ffi {
                 #[swift_bridge(swift_name = "FfiFoo", swift_repr = "class")]
+                #[derive(Copy, Clone)]
```

Comment:
Can we remove this derive?

Reply:
I'm actually testing here whether `#[swift_bridge]` and `#[derive]` work together here. I think it's a good test to have, so I'll just leave this test as is and duplicate it to check if they work ok together

Reply:
[ead0af9](https://github.com/chinedufn/swift-bridge/pull/198/commits/ead0af99013d5cafb5e2cb8d0da6ba7a191d8211)